storage/allocator.go

```diff
@@ -41,6 +42,10 @@ const (
 	// is within rebalanceFromMean of the mean, it is considered a
 	// viable target to rebalance to.
 	rebalanceFromMean = 0.025 // 2.5%
+	// rebalanceShouldRebalanceChance represents a chance that an individual
```

Comment:
Choosing a random replica is something we already get implicitly from the scanner; the scanner is constantly iterating through replicas, eventually "shouldRebalance" will return true for one of them. While this method is indirect, we essentially get it for free; otherwise, we would need another thread somewhere polling for a rebalance opportunity.